internal/controller/mantlebackup_controller.go

```diff
@@ -765,6 +859,107 @@ func (r *MantleBackupReconciler) export(
 	return ctrl.Result{}, nil
 }
 
+func (r *MantleBackupReconciler) startImport(
+	ctx context.Context,
+	backup *mantlev1.MantleBackup,
+	target *snapshotTarget,
+) (ctrl.Result, error) { //nolint:unparam
+	if result, err := r.checkExportDataAlreadyUploaded(ctx, backup); err != nil || !result.IsZero() {
```

Comment:
```suggestion
	if result, err := r.isExportDataAlreadyUploaded(ctx, backup); err != nil || !result.IsZero() {
```

It's to clarify this function returns true if the data has already been uploaded.

Reply:
Fixed in [601de02](https://github.com/cybozu-go/mantle/pull/53/commits/601de02dd7d68d2a66a6f8612dda43a00bd2705f)